src/test_model.py

```diff
@@ -0,0 +1,29 @@
+from gurobipy import *
+
+try:
```

Comment:
Why is there a try-catch clause on the whole code? Do you know what kind of error you get?

Reply:
It is because you can have several Errors. This try catch was just illustrating the Error of the Gurobi module.